src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -114,6 +114,13 @@
 				@click="promoteToModerator">
 				{{ t('spreed', 'Promote to moderator') }}
 			</ActionButton>
+			<ActionButton v-if="isEmailActor"
```

Comment:
But logged in users have pins too, when SIP is enabled. So also show the option in that case?

Reply:
I did a test with users who have emails. If I add those to a SIP-enabled conversation they never receive an invitation email as they are not ACTOR_EMAILS.

Should we make the resend feature still send an email to those people who never received one in the first place ? This means moving the scope from ACTOR_EMAILS to all user actors who have an email address set.

Reply:
Well I think it would be nice, as they might not have their nextcloud log in at hand, but emails + dialin would be possible.
But well then lets keep it for now, as I forgot email addresses are optional 🙈